EternalModLoader/BlangDecrypt.cs

```diff
@@ -0,0 +1,217 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace EternalModLoader
+{
+    /// <summary>
+    /// BlangCrypt class
+    /// </summary>
+    internal class BlangCrypt
+    {
+        /// <summary>
+        /// Encrypts or decrypts a blang file
+        /// </summary>
+        /// <param name="fileData">byte array contaning a blang file bytes</param>
+        /// <param name="internalPath">blang file's internal path</param>
+        /// <param name="decrypt">bool indicating if we wanna encrypt or decrypt</param>
+        /// <returns>int indicating success or failure</returns>
+        public static int IdCrypt(ref byte[] fileData, string internalPath, bool decrypt)
+        {
+            string keyDeriveStatic = "swapTeam\n";
+
```

Comment:
Please join the previous and next lines together and separate them from the condition with an extra line break.